### PR TITLE
Create Millennia's portfolio home page

### PR DESCRIPTION
**Pull request by hilakey**
I would like feedback on my code, if the format style is okay.
- Added second page to portfolio.
- Added images for the gallery section.

Also, should anything else be added, or could be modified to suit the purpose of the first project?

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -4,15 +4,60 @@
...
+            <p>Here's my gallery of my favorite art pieces from the museums I've visited!</p>
+            <img src="images/art1.JPG" width="300" height="400">
+            <img src="images/art2.JPG" width="380" height="400">
+            <img src="images/art3.JPG" width="300" height="400">
+            <img src="images/art4.JPG" width="300" height="400">
+
+            <p> And I have a chihuahua named Sonic</p>
+            <a href="images/sonic.jpg"><img src="images/sonic.jpg" width="300" height="400"></a>
+            <a href="images/sonic2.PNG"><img src="images/sonic2.PNG" width="300" height="400"></a>
+
```
Comment (gagomez) [inline portfolio/src/main/webapp/index.html]: The indenting here seems off. 

One thing you could do if you want to group these as a single element is you could wrap them in a `<div>`. For example:
```
<h2>Gallery</h2>
<div>
  <p>...
  <img>...
</div>
```

In a future stand up we can talk about components!

File: portfolio/src/main/webapp/myportfoliopage2.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        The ones I listed here are some of the games that really impacted me with the experience that I got from them,<br>
+        and I think that if you're a gamer, you might find something you may want to try here!</p>
+       
+        <p> Resident Evil 2 Remake </p>
+        <a href="images/re2.jpg"><img src="images/re2.jpg" width="300" height="300" alt="Resident Evil 2 Remake"></a><
+        <p> Final Fantasy VII Remake (My #1 game)</p>
+        <a href="images/FF7R.jpg"><img src="images/FF7R.jpg" width="500" height="300" alt="Final Fantasy VII Remake"></a>
+        <p> Final Fantasy XIII Trilogy</p>
+        <a href="images/FF13.jpg"><img src="images/FF13.jpg" width="500" height="250" alt="Final Fantasy 13 Trilogy"></a>
+       <p> HALO</p>
```
Comment (gagomez) [inline portfolio/src/main/webapp/myportfoliopage2.html]: Indentation is just a bit off. Off by one space.

**Review by gagomez — APPROVED** [review]

File: portfolio/src/main/webapp/myportfoliopage2.html (hunk 2 above)
Comment (hilakey) [reply portfolio/src/main/webapp/myportfoliopage2.html]: Thank you for catching that! Just fixed the formatting.

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -12,17 +12,22 @@
...
+
+// When selected by the user the button will generate a random link to watch any of the animes listed in the array
+
+function randomAnime(){
+    const animeList = ["https://www.crunchyroll.com/jojos-bizarre-adventure", "https://www.crunchyroll.com/yu-gi-oh-zexal", "https://www.crunchyroll.com/toradora"
+    , "https://www.crunchyroll.com/fruits-basket", "https://www.crunchyroll.com/dragon-ball-super", "https://www.crunchyroll.com/my-hero-academia", "https://www.crunchyroll.com/sword-art-online", "https://www.netflix.com/title/80241960"];
+
+    const animePick = animeList[Math.floor(Math.random() * animeList.length)];
+    window.open(animePick);
+
```
Comment (sasymer) [inline portfolio/src/main/webapp/script.js]: Maybe could get rid of these extra spaces

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -1,9 +1,35 @@
...
+p{
+    font-family:Arial, Helvetica, sans-serif;
+    color: black;
+    font-weight: normal;
+}
+
+h1{
+    color: white;
+    background-color: pink;
+     border-width: 4px;
```
Comment (sasymer) [inline portfolio/src/main/webapp/style.css]: Indentation has one extra space.

**Review by sasymer — APPROVED** [review]
I left a couple comments just about spacing/indentation-looks good to me! Hopefully the comments will work and go through to you.

File: portfolio/src/main/webapp/style.css (hunk 4 above)
Comment (hilakey) [reply portfolio/src/main/webapp/style.css]: Thank you Sanna, I just reformatted it!

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (hilakey) [reply portfolio/src/main/webapp/script.js]: Done!

**Pull request merged**